table/block_based/block_builder.h

```diff
@@ -72,7 +72,7 @@ class BlockBuilder {
   // Returns an estimated block size after appending key and value.
   size_t EstimateSizeAfterKV(const Slice& key, const Slice& value) const;
 
-  // Return true iff no entries have been added since the last Reset()
+  // Return true if no entries have been added since the last Reset()
```

Comment:
I guess the original version of the comment uses short form for "if and only if".
Whether that's what the code does, it hard to say with the context.
Ref https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Oh, I'm sorry I forgot that `iff` has a unique meaning. I checked the code corresponding to this comment, and it is more rigorous to use `iff`. thanks.